packages/scrum-server/src/api/components/userStory/service.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+// The service class acts like a wrapper for the database. Here we read and write data to the database. Furthermore, we can implement caching for example.
+const i18n = require('i18n');
+const UserStory = require('./model');
+const ObjectId = require('mongoose').Types.ObjectId;
+
+const save = async jsonData => {
+  try {
+    const newUserStory = new UserStory(jsonData);
+    const docUserStory = await newUserStory.save();
+    return {
+      data: docUserStory,
+    };
+  } catch (e) {
+    console.log(e);
+    throw Error(i18n.__('serviceSaveError'));
+  }
+};
+
+const find = async objectId => {
+  if (!objectId || !ObjectId.isValid(objectId)) {
+    throw Error(i18n.__('serviceInvalidId'));
+  }
+  const userStory = await UserStory.findOne({ _id: objectId });
+  if (!userStory) {
+    throw Error(i18n.__('serviceFindError'));
```

Comment:
Shall we create error constants ?

Reply:
Lets discuss that, i think yes makes sense, but will that have to be dynamic? for the i18n?